source/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -0,0 +1,32 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
+    name = "http2_platform_impl_lib",
+    hdrs = [
+        "http2_arraysize_impl.h",
```

Comment:
Why do we have this enumeration of headers as well as the one in `quiche.BUILD`?